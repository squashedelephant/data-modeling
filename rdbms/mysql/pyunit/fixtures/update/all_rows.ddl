CREATE TABLE data(id INT NOT NULL AUTO_INCREMENT, name VARCHAR(20), PRIMARY KEY (id));
