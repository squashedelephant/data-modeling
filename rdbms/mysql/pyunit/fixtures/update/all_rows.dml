INSERT INTO data(name) VALUES ('Alpha'), ('Beta'), ('Gamma'), ('Delta'), ('Epsilon');
UPDATE data SET name='Omega' WHERE name='Alpha';
UPDATE data SET name='Psi' WHERE name='Beta';
UPDATE data SET name='Chi' WHERE name='Gamma';
UPDATE data SET name='Phi' WHERE name='Delta';
UPDATE data SET name='Upsilon' WHERE name='Epsilon';
