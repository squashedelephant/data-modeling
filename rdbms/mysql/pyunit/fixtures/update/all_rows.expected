Omega
Psi
Chi
Phi
Upsilon
